### ISSUE TITLE
Some minor accessibility improvements in using git.

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#126382]
Issue Type: <b>Feature Request</b>

cc: @isidorn

When performing an operation in git, the orca, the screen reader I use to access the VSCode, does not inform me about the result of the operation. This happens for example when I commit, pull, push and so on. I never know if the operation ended and what the result was.
I also can't easily identify if VSCode is asking for a login to access the repository.
Maybe the VSCode is providing this information and the orca just isn't reading it.

VS Code version: Code - Insiders 1.58.0-insider (e1161be26874f0d39a5b9cdeb2c5fbfd4f83bb80, 2021-06-14T05:13:48.044Z)
OS version: Linux x64 5.12.10-arch1-1
Restricted Mode: No

**Comment by isidorn** [comment]
@jvesouza thanks for the feedback!
In general vscode does not announce when some command successfully finished. Doing this as a special case for SCM commands would be a bit weird. 
I think we should do the opposite approach, be 100% sure that if the command fails that the user is aware of it. The only missing piece would be how to know if the operation is still in progress - and for that we are actually using `progress` api which should be read out by the screen reader if the progress is long enough. I am not sure if this work well in practice, but in theory it should work (I tried it with Orca).

@jvesouza What do you think about this approach? 

As for the login not being read, that sounds like an issue we should fix, do you have steps for this which we could follow and reproduce?

**Comment by yume-chan** [comment]
> Doing this as a special case for SCM commands would be a bit weird.

There is a long-existed setting `git.showPushSuccessNotification`, which when set to `true`, will show a notification after `git.push` command succeeded. But it doesn't show notification for other SCM operations, even the `git.sync` command.

**Comment by jvesouza** [comment]
> 
> @jvesouza What do you think about this approach?

To be honest, I don't really like this approach, but I fully understand that it's a matter of point of view. I always like to be informed of the results of operations.
> 
> As for the login not being read, that sounds like an issue we should fix, do you have steps for this which we could follow and reproduce?

As for the login issue, it doesn't seem to be directly related to git but rather a general issue. I found for example that the same problem happens when I try to log in to a remote server using the remote ssh extension. If the connection requires a login / password, the orca doesn't read the prompts unless I press shift+tab and tab.

**Comment by isidorn** [comment]
@jvesouza can you please open a new issue for the login problem and ping me @isidorn on it
As for the git no success notification, can you please try the setting 
```
	"git.showPushSuccessNotification": true,
```

We can maybe change the default of that setting when a screen reader is detected.
Or introduce a more general setting that would cover all the git operations.

**Comment by isidorn** [comment]
No, I do not think there is anything that should be done that is specific for `scm` thus let's close this.
In general I think we should introduce some smartness when a screen reader is detected that we have a way to notify users when a command completes. Just that we keep that in mind of the future as a general workbench thing.
